README.md

```diff
@@ -0,0 +1,31 @@
+# matrix-auth-plugin
+Matrix-based authorization strategies for Jenkins
+
+Sample pipeline (assumes matrix based security enabled on jenkins):
+
+    properties ([
+  		  [$class: 'AuthorizationMatrixProperty',
```

Comment:
If you add a dependency on the `structs` plugin you could add a `@Symbol` and make this look like, say,

```groovy
properties([authorizationMatrix(['hudson.model.Item.Build:admin','hudson.model.Item.Build:user'])])
```